e2e/support/helpers/component-testing-sdk/component-embedding-sdk-helpers.tsx

```diff
@@ -60,15 +61,18 @@ export function mountSdkContent(
   cy.intercept("GET", "/api/user/current").as("getUser");
 
   cy.mount(
-    <MetabaseProvider
-      {...sdkProviderProps}
-      authConfig={{
-        ...DEFAULT_SDK_AUTH_PROVIDER_CONFIG,
-        ...sdkProviderProps?.authConfig,
-      }}
-    >
-      {children}
-    </MetabaseProvider>,
+    <React.StrictMode>
+      <MetabaseProvider
+        {...sdkProviderProps}
+        authConfig={{
+          ...DEFAULT_SDK_AUTH_PROVIDER_CONFIG,
+          ...sdkProviderProps?.authConfig,
+        }}
+      >
+        {children}
+      </MetabaseProvider>
+      ,
```

Comment:
```suggestion
```
Is this an extra comma that gets rendered?

Reply:
@npretto wow, good eye!